Automattic/jetpack
# E2E Tests: Add plan upgrade flow

Partial rewrite of https://github.com/Automattic/jetpack/pull/15745

Add new E2E test that covers plan upgrade flow. 

#### Changes proposed in this Pull Request:

* Support sandbox cookie for _all_ WPCOM requests
* Add plan upgrade flow

#### Jetpack product discussion


n/a
#### Does this pull request change what data or activity we track or use?


n/a
#### Testing instructions:






* Make sure tests are green in CI.
* Run test locally, make sure it passes as well: `E2E_DEBUG=true PUPPETEER_HEADLESS=false yarn test-e2e tests/e2e/specs/connection.test.js -t "In-place upgrading"`
*

#### Proposed changelog entry for your changes:


* n/a
